test/tests/jobs/FinishJobTest.cpp

```diff
@@ -353,4 +354,26 @@ struct FinishJobTest : tpunit::TestFixture {
         time_t nextRunTime = mktime(&tm2);
         ASSERT_EQUAL(difftime(nextRunTime, createdTime), 3600);
     }
+
+    // Finish job in RUNQUEUED state
+    void inRunqueuedState() {
+        // Create a job
+        SData command("CreateJob");
+        command["name"] = "job";
+        command["retryAfter"] = "+1 SECOND";
+        STable response = tester->executeWaitVerifyContentTable(command);
+        string jobID = response["jobID"];
+
+        // Get the job
+        command.clear();
+        command.methodLine = "GetJob";
+        command["name"] = "job";
+        tester->executeWaitVerifyContent(command);
```

Comment:
In previous tests you're asserting that you're getting the right info in the response, why not here?

Reply:
This is a test for `FinishJob` functionality.  A test for `GetJob` should verify the return values of `GetJob`